client/src/app/site/assignments/modules/assignment-poll/components/assignment-poll-meta-info/assignment-poll-meta-info.component.html

```diff
@@ -0,0 +1,49 @@
+<ng-container *ngIf="poll">
+    <!-- Candidates -->
+    <small class="meta-info-left subtitle" *ngIf="poll.options?.length && showCandidates">
+        <div>
+            {{ 'Candidates' | translate }}:
+            <ol>
+                <li *ngFor="let option of poll.options">{{ getOptionTitle(option) | translate }}</li>
+            </ol>
+        </div>
+        <div *ngIf="hasGlobalOption">
+            {{ 'Options' | translate }}:
+            <ul>
+                <li *ngIf="poll.global_yes">{{ pollPropertyVerbose.global_yes | translate }}</li>
+                <li *ngIf="poll.global_no">{{ pollPropertyVerbose.global_no | translate }}</li>
+                <li *ngIf="poll.global_abstain">{{ pollPropertyVerbose.global_abstain | translate }}</li>
+            </ul>
+        </div>
+    </small>
+
+    <div class="meta-info-right subtitle">
+        <!-- Groups who can vote -->
+        <small *ngIf="poll.groups && poll.type && poll.type !== 'analog'">
+            {{ 'Entitled to vote' | translate }}:
+
+            <span *ngFor="let group of poll.groups; let i = index">
+                {{ group.getTitle() | translate }}<span *ngIf="i < poll.groups.length - 1">, </span>
+            </span>
+            <br />
+        </small>
+
+        <!-- Voting method -->
+        <small *ngIf="poll.pollmethod">
+            <span> {{ 'Voting method' | translate }}: {{ poll.pollmethodVerbose | translate }} </span>
+            <!-- amount of votes -->
+            <span *ngIf="poll.isMethodY || poll.isMethodN">
+                ({{ poll.votes_amount }}
+                <span *ngIf="poll.votes_amount === 1">{{ 'Vote' | translate }}</span>
```

Comment:
as shown in the screenshot, the translated word for "vote" is wrong.

Reply:
As far as I know there is the possibility to give a context while translating strings, so one word (like this one) can be given two translations with different contexts. We never have used it, but here this would be the best solution (isntead of writing-around this string). But it might be a bit more complex since we never have used it.

As for the webpage part (not extracting), there is the possibility to give params. I think this is the context I meant: https://github.com/ngx-translate/core#api

Reply:
yes, but our setup cannot do that since we are using po, pot and json instead of the recommended way to handle translations by ngx-translate

Reply:
> We never have used it, but here this would be the best solution (isntead of writing-around this string). But it might be a bit more complex since we never have used it.

and, furthermore, we will not take this effort. Since angular 8 ngx-translate can be considered legacy, we'd rather remove it and switch to angulars build in functions.

Reply:
ok, then please hide again the case x ==1... sorry

Reply:
> and, furthermore, we will not take this effort. 

Yes I think this is rational. (I added the comment, just to show that there *are* ways, but the path to the solutions might take some major refactoring)